draft-birkholz-scitt-receipts.md

```diff
@@ -252,57 +251,49 @@ ProofElement = [
 ]
 
 LeafInfo = [
-    prefix: bstr,
-    type: LeafEntryType,
-    data: LeafReceiptData
+    writeset_hash: bstr,
```

Comment:
Asking out of curiosity more than anything else, but this is "write_set_digest" in CCF, is there a reason why it's been renamed?